frontend/src/App.tsx

```diff
@@ -60,6 +61,7 @@ const App = () => {
               draggable
               pauseOnHover
             />
+            <ModalContainer />
```

Comment:
is this part of the story or a tad of tech-debt

Reply:
What would happen if there are multiple modals being stacked? I think the Lease->Deposit->Add Contact workflow will do that

Reply:
it is not possible to stack multiple modals, they would have to just be floating popup windows which is not what this container was built for. If you want multiple stacking popups you'll need to do things the old way.